[tgui] Update portfile.cmake

tGstep opened the pull request:
Update to the 0.8.5 branch.

tGstep committed:
Update portfile.cmake

Update to the 0.8.5 branch.

Rastaban commented:
Thanks for the submission! Don't forget to update the CONTROL file version too.

Rastaban commented:
I see you have a separate PR for the CONTROL file change (#6193).  Can you add that change into this PR instead?  It helps the git version history to be cleaner if it is done in the same PR.

tGstep committed:
Update CONTROL

msftclas commented:
[![CLA assistant check](https://cla.opensource.microsoft.com/pull/badge/not_signed)](https://cla.opensource.microsoft.com/Microsoft/vcpkg?pullRequest=6192) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.opensource.microsoft.com/Microsoft/vcpkg?pullRequest=6192) before we can accept your contribution.<br/><br/>:x: tGstep [sign now](https://cla.opensource.microsoft.com/Microsoft/vcpkg?pullRequest=6192)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.microsoft.com/check/Microsoft/vcpkg?pullRequest=6192) it.</sub>

NancyLi1013 commented:
Hi @tGstep, here are test results from the current CI system, please let us know if you need any help.

|x64-windows|master| test |notes
|-----------|------|------|-----
|tgui       |Pass  |Fail  |**Regression**

|x64-windows-static|master| test |notes
|------------------|------|------|-----
|tgui              |Pass  |Fail  |**Regression**

|x64-osx|master| test |notes
|-------|------|------|-----
|tgui   |Pass  |Fail  |**Regression**

|x86-windows|master| test |notes
|-----------|------|------|-----
|tgui       |Pass  |Fail  |**Regression**

Rastaban committed:
[tgui]

Combine download/extract functions into vcpkg_from_github.  Remove nanosvg published headers.
